docs/api/generator/gcr.md

```diff
@@ -0,0 +1,30 @@
+GCRAccessToken creates a GCP Access token that can be used to authenticate with GCR in order to pull OCI images.
+
+You must specify the `spec.projectID` in which GCR is located.
```

Comment:
```suggestion
GCRAccessToken creates a GCP Access token that can be used to authenticate with GCR in order to pull OCI images. You won't need any extra permissions to request for a token, but the token would only work against a GCR if the token requester (service Account or WI) has the appropriate access

You must specify the `spec.projectID` in which GCR is located.
```

Reply:
I found this a bit confusing while playing with it, so maybe a clarification would be nice